index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Apple</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.9.0/css/all.css"
+      integrity="sha384-i1LQnF23gykqWXg6jxC2ZbCbUMxyw5gLZY6UiUS98LYV5unm8GWmfkIS6jqJfb4E"
+      crossorigin="anonymous" />
+    <link rel="stylesheet" href="assets/css/style.css" />
+  </head>
+  <body>
+    <main class="site__header main">
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav-logo">
+            <a href=""><img src="assets/img/5d34bc9073b21.png" alt=""/></a>
+          </li>
+          <li><a href="#">Store</a></li>
+          <li><a href="#">Mac</a></li>
+          <li><a href="#">iPod</a></li>
+          <li><a href="#">iPhone</a></li>
+          <li><a href="#">iPad</a></li>
+          <li><a href="#">iTunes</a></li>
+          <li class="nav-last"><a href="#">Support</a></li>
+          <li>
+            <div class="search-bar">
+              <!-- <i class="fas fa-search"></i> -->
+              <input type="search" name="search" id="search" />
+            </div>
+          </li>
+        </ul>
+      </nav>
+
+      <section class="header">
+        <a class="header__top" href="#">
+          <div class="top-image"></div>
+        </a>
+        <a class="header__center" href="#">
+          <div class="center-image"></div>
+        </a>
+        <a class="header__bottom" href="#">
+          <div class="bottom-image"></div>
+        </a>
+      </section>
+    </main>
+
+    <aside>
+      <ul class="aside-container">
+        <li class="aside__box box-1"><a href="#"></a></li>
+        <li class="aside__box box-2"><a href="#"></a></li>
+        <li class="aside__box box-3"><a href="#"></a></li>
+        <li class="aside__box box-4"><a href="#"></a></li>
+      </ul>
+    </aside>
+
```

Comment:
These 4 boxes should go within the footer. They should be `<a>` tags containing `<div>` tags, those `<div>` tags should have the images set as backgrounds.